sphinx-autobuild is broken with addition of delay parameter to livereload/watcher

Changes to python-reload at https://github.com/lepture/python-livereload/commit/d73e612eb949c3b6748b8939b95399cd34946b8f break sphinx-autobuild.

The error I see is -

```
sphinx-autobuild -b html docs/source docs/build/html
Traceback (most recent call last):
  File "/home/manas/code/foo/virtualenv/bin/sphinx-autobuild", line 11, in <module>
    sys.exit(main())
  File "/home/manas/code/foo/virtualenv/local/lib/python2.7/site-packages/sphinx_autobuild/__init__.py", line 208, in main
    server.watch(srcdir, SphinxBuilder(outdir, build_args, ignored))
  File "/home/manas/code/foo/virtualenv/local/lib/python2.7/site-packages/livereload/server.py", line 173, in watch
    self.watcher.watch(filepath, func, delay)
TypeError: watch() takes exactly 3 arguments (4 given)
make: *** [.livedocs] Error 1
```

The change on the sphinx side would have to be in the function https://github.com/GaretJax/sphinx-autobuild/blob/develop/sphinx_autobuild/__init__.py#L75

If possible will try to submit a patch.

--- a/requirements.txt
+++ b/requirements.txt
@@ -3,4 +3,4 @@ argh>=0.24.1
 pathtools>=0.1.2
 PyYAML>=3.10
 tornado>=3.2
-livereload>=2.2.0
+livereload>=2.3.0

--- a/sphinx_autobuild/__init__.py
+++ b/sphinx_autobuild/__init__.py
@@ -59,6 +59,9 @@ class LivereloadWatchdogWatcher(object):
         self.filepath = None  # Accessed by LiveReloadHandler's watch_task
                               # method. When set to a boolean false value,
                               # everything is reloaded in the browser ('*').
+        self._changes = []  # Accessed by Server's serve method to set
+                            # reload time to 0 in LiveReloadHandler's
+                            # poll_tasks method.
 
     def set_changed(self):
         self._changed = True
@@ -68,11 +71,16 @@ class LivereloadWatchdogWatcher(object):
         Called by LiveReloadHandler's poll_tasks method. If a boolean true
         value is returned, then the waiters (browsers) are reloaded.
         """
+        if self._changes:
+            return self._changes.pop()
+
+        action_file = None
         if self._changed:
             self._changed = False
-            return self._action_file or True  # TODO: Hack (see above)
+            action_file = self._action_file or True  # TODO: Hack (see above)
+        return action_file, None
 
-    def watch(self, path, action):
+    def watch(self, path, action, _):
         """
         Called by the Server instance when a new watch task is requested.
         """

